docs/examples.rst

```diff
@@ -141,3 +141,28 @@ One form of running this example could be:
 .. explicit external hyperlink targets
 
 .. _examples: https://github.com/LLNL/UnifyFS/tree/dev/examples/src
+
+Transfer API
+------------
+
+UnifyFS has a transfer API to move files from UnifyFS to external storage (or from external storage into UnifyFS).  The transfer functionality can be invoked by using the "transfer" application in the examples directory as show in the following example:
+
+.. code-block:: Bash
+
+    $ srun -N4 -n4 transfer-static /unifyfs/file1 /scratch/mydir/file1
+
+(assuming that /unifyfs/file1 is a file you've written within the UnifyFS file space by an application.)  To use the transfer API functions directly in a C program, use the following as a template:
+
+.. code-block:: C
+
+    #include <unifyfs.h>
+    ...
+    // in an MPI context
+    if (rank_to_copy == my_rank) {
+            ret = unifyfs_transfer_file_serial(srcpath, dstpath);
+            if (ret) {
+                fprintf(stderr,"unifyfs copy failed (%d: %s)", ret, strerror(ret));
+            }
+        }
```

Comment:
If we keep this if statement, are these lines are indented one too many levels or at least the closing brace for the "if"?